app.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const healthCheckController = require("./db/controllers/health-check-controller.js");
+const { getTopics } = require("./db/controllers/topics-controller");
+const data = require("./db/data/development-data");
+
+const app = express();
+app.use(express.json());
```

Comment:
We should not have this here as it is not needed at the moment, you can put it back one your request has a body 🙂